src/views/Viewer.vue

```diff
@@ -270,7 +271,7 @@ export default {
 			isSidebarShown: false,
 			isFullscreenMode: false,
 			canSwipe: true,
-			isStandalone: !(OCA && OCA.Files && 'fileActions' in OCA.Files),
+			isStandalone: !(OCA && OCA.Files && 'fileActions' in OCA.Files), // FIXME this probably needs some adjustment
```

Comment:
Left this comment intentionally, as i think we'll need to adjust this as soon as the old code bits are dropped and public share links are switched as well. We'll need a different approach then